### PR TITLE
minor changes on authorization part

### PR DESCRIPTION
**Pull request by karesti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,17 +72,25 @@ All of the cli commands defined in this file are executed before the server is s
...
+user create "deployer" -p "secret2" 
+user create "Rigoberta Baldini" -p "secret3" -g monitor
 ```
 
-To run the image using a local `idenitities.batch`, execute:
+To run the image using a local `identities.batch`, execute:
 
 ```bash
-docker run -v $(pwd):/user-config -e IDENTITIES_BATCH="/user-config/identities.batch" infinispan/server
+docker volume create user-config
```
Comment (ryanemerson) [inline README.md:92]: This line isn't necessary, the `-v` flag with `docker run ...` automatically creates the volume.

File: README.md [hunk 2]
```diff
@@ -63,8 +63,6 @@ When connecting a HotRod client to the image, the following SASL properties must
 ```properties
 infinispan.client.hotrod.auth_username=admin
 infinispan.client.hotrod.auth_password=changme
-infinispan.client.hotrod.auth_realm=default
-infinispan.client.hotrod.auth_server_name=infinispan
```
Comment (ryanemerson) [inline README.md:67]: Are these configured by default now?

File: README.md [hunk 3]
```diff
@@ -107,6 +113,7 @@ discover cluster members. To enable this, we must provide the `jgroups.dnsPing.q
 To utilise the tcp stack with DNS_PING, execute the following config:
 
 ```bash
+docker volume create user-config
```
Comment (ryanemerson) [inline README.md:116]: Same as https://github.com/infinispan/infinispan-images/pull/201/files#r1005432401

File: README.md (hunk 2 above)
Comment (karesti) [reply README.md:67]: The defaults are 

```java
  public static final String DEFAULT_REALM = "default";
   public static final String DEFAULT_SERVER_NAME = "infinispan";
   public static final String DEFAULT_MECHANISM = "SCRAM-SHA-512";
```

the AuthenticationConfigurationBuilder uses those if not provided

File: README.md (hunk 1 above)
Comment (karesti) [reply README.md:92]: it was not working for me, but maybe for anothe reason I will recheck

**Comment by karesti** [comment]
@ryanemerson changes done

**Comment by ryanemerson** [comment]
Thanks @karesti

**Pull request merged**